tests/api_v2_tests/test_auth.py

```diff
@@ -0,0 +1,53 @@
+"""
+tests for user authentication
+"""
+import json
+
+DATA = [{'first_name':'Simon', 'last_name': 'Mbugua',
+         'email': 'simon@email.com', 'password':"testpassword"},
+        {'email':'simon@email.com', 'password':"testpassword"},
+        {'email': 'swwee@mail.com', 'password':"testpassword"}
+       ]
+
+
+def test_signup(test_client):
+    """
+    test that accounts can be created
+    """
+
+    response = test_client.post('/auth/signup', data=json.dumps(DATA[0]),
+                                content_type='application/json')
+
+    assert response.status_code == 201
+
+def test_signup_twice(test_client):
```

Comment:
Hey have you thought what would happen if the signup functionality is buggy but still returns status code 201?